### PR TITLE
Create recommendations from Eric for San Diego

### LINKED ISSUE #3: Create a new file for githubteacher's hometown (San Diego, CA)
## San Diego, CA
- [x] Create a branch
- [x] Add commits (make changes)
- [x] Open a Pull Request
- [x]  Collaboration
- [x] Seek approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by githubteacher**
closes #3

### CODE REVIEW

File: places/githubteacher-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# San Diego, CA
+
+### Places to Eat
+- City Tacos
```
Comment (hectorsector) [inline places/githubteacher-sandiego.md:4]: @githubteacher I like 🍣 -- do you have any recommendations for that?

File: places/githubteacher-sandiego.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# San Diego, CA
+
+### Places to Eat
+
+### Places to Avoid
+
+### Places to Go
```
Comment (hectorsector) [inline places/githubteacher-sandiego.md:10]: I hear SeaWorld is awesome.

File: places/githubteacher-sandiego.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# San Diego, CA
+
+### Places to Eat
+- City Tacos
+- Sushi N' Joy
```
Comment (hectorsector) [inline places/githubteacher-sandiego.md:5]: Oh, I remember this place! Awesome. 👍 definitely add

File: places/githubteacher-sandiego.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+### Places to Avoid
+- Sea World
+
+### Places to Go
+- Coronado
+
+### Fun Facts
+- It's 90 degrees. We don't have fall.
+- Top Gun was filmed here.
+- We need water.
```
Comment (hectorsector) [inline places/githubteacher-sandiego.md:16]: I don't know what this means. Could you clarify @githubteacher?

**Review by hectorsector — DISMISSED** [review]
This is almost ready to go, but could you 👀 my recent comments before you 🚢?

**Comment by githubteacher** [comment]
👍  This looks good to 🚢

**Review by hectorsector — DISMISSED** [review]
I guess I have no choice but to 👍

**Pull request merged**